google/api_core/client_logging.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import json
+import re
+import os
+
+_LOGGING_INITIALIZED = False
+_BASE_LOGGER_NAME = "google"
+# TODO(https://github.com/googleapis/python-api-core/issues/760): Update Request / Response messages.
+REQUEST_MESSAGE = "Sending request ..."
+RESPONSE_MESSAGE = "Receiving response ..."
```

Comment:
```suggestion
REQUEST_MESSAGE = "Sending request "
RESPONSE_MESSAGE = "Receiving response "
```
(space at the end intentional if we want to later append the RPC name)